src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -4135,12 +4136,57 @@ def agg(
         numeric_only = kwargs.get("numeric_only", False)
         # Call fallback if the aggregation function passed in the arg is currently not supported
         # by snowflake engine.
-        if not check_is_aggregation_supported_in_snowflake(func, kwargs, axis):
-            ErrorMessage.not_implemented(
-                f"Aggregate function {func} with parameters "
-                + ", ".join([f"{k}={v}" for k, v in kwargs.items()])
-                + " not supported yet in Snowpark pandas."
-            )
+        # If we are using Named Aggregations, we need to do our supported check slightly differently.
+        uses_named_aggs_df = (
+            is_dict_like(func)
```

Comment:
can you give an example here about how the data structure format look like when checking if it whether uses_named_aggs_df?